packages/graphql-tool-utilities/package.json

```diff
@@ -25,9 +27,6 @@
     "pretest": "yarn build",
     "prepublishOnly": "yarn build"
   },
-  "devDependencies": {
-    "typescript": "^2.8.3"
```

Comment:
this is probably just a relic of old times. should not be necessary with the root level `devDependency`